test/test-utils.js

```diff
@@ -331,3 +337,123 @@ export const adminCreateCategory =
     }
     return res;
   };
+export const createVoucherType = (chain) => async (description, duration) => {
+  const VOUCHER_HUB_ABI = {
+    inputs: [
+      {
+        internalType: 'string',
+        name: 'description',
+        type: 'string',
+      },
+      {
+        internalType: 'uint256',
+        name: 'duration',
+        type: 'uint256',
+      },
+    ],
+    name: 'createVoucherType',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  };
```

Comment:
abi must be an array of abis fragments

```suggestion
  const VOUCHER_HUB_ABI = [{
    inputs: [
      {
        internalType: 'string',
        name: 'description',
        type: 'string',
      },
      {
        internalType: 'uint256',
        name: 'duration',
        type: 'uint256',
      },
    ],
    name: 'createVoucherType',
    outputs: [],
    stateMutability: 'nonpayable',
    type: 'function',
  }];
```